src/components/hand-controls.js

```diff
@@ -156,25 +156,31 @@ module.exports.Component = registerComponent('hand-controls', {
     return controllerId === 'OpenVR Gamepad';
   },
 
+  isOculusTouchController: function () {
```

Comment:
right now `isViveController` isn't used, I will remove it.  There are probably better ways to check whether Oculus Touch, since `hand-controls` injects `oculus-touch-controls` and so should be able to check its `controllerPresent` -- but it seemed convenient to make a function for clarity.  If placed in utils, it won't have the entity context (or the assumption that it's `hand-controls`) so some other way to confirm that the entity must be at least `tracked-controls` might be needed.